mne/beamformer/_dics.py

```diff
@@ -189,11 +193,14 @@ def make_dics(info, forward, csd, reg=0.05, noise_csd=None, label=None,
                         (freq, i + 1, n_freqs))
 
         Cm = csd.get_data(index=i)
+
+        # XXX: Weird that real_filter happens *before* whitening, which could
+        # make things complex again...?
```

Comment:
Agree, @wmvanvliet can you double check?

Reply:
@larsoner what do we do with this? Keep as is and investigate what it would change in new PR?